messaging-clustering-singleton/charts/helm.yaml

```diff
@@ -0,0 +1,6 @@
+build:
```

Comment:
Not needed? Considering this is not OS compatible quickstart...

Reply:
done